openedx/event-routing-backends
# feat: allow to route events in sync

## Description

This PR creates a handler to process tracking log from the event bus.

- Update the event router to include a field to define when to process the events in celery or in the same thread.
- Update the setting to use the event bus by default.

## Testing instructions
- Install openedx-events from the mentioned PR
- Update your `EVENT_TRACKING_BACKENDS` settings to use the engine `eventtracking.backends.event_bus.EventBusRoutingBackend` and update the caliper and xapi backends to use: `event_routing_backends.backends.sync_events_router.SyncEventsRouter`
- Install event-tracking from this PR https://github.com/openedx/event-tracking/pull/246
- Configure the event bus
- Run: ` tutor dev exec lms ./manage.py lms consume_events -t analytics -g event_routing_backends --extra '{"consumer_name": "aspects"}'`
- Install, enable and initialize tutor-contrib-aspects
- Verify the xAPI statements are sent to Ralph
